lib/iiif_print/blacklight_iiif_search/annotation_decorator.rb

```diff
@@ -28,19 +29,30 @@ def canvas_uri_for_annotation
       # @return [String]
       def coordinates
         return default_coords if query.blank?
+
+        sanitized_query = sanitize_query
         coords_json = fetch_and_parse_coords
-        return default_coords unless coords_json && coords_json['coords']
-        sanitized_query = query.match(additional_query_terms_regex)[1].strip
+
+        coords_check_result = check_coords_json_and_properties(coords_json, sanitized_query)
```

Comment:
Thinking about moving the guard condition of the original method outside of the original method, as it has a surprising return value (based on the given method name).

`return derived_coords_json_and_properties(coords_json, sanitized_query) unless coords_json['coords']`